src/resolvers/get-model.ts

```diff
@@ -365,7 +386,13 @@ function getModelFromMakeDescription(make: Make, description: string): string |
       if (description.match(/gle/i)) {
         return Model[make].GLE
       }
-      if (description.match(/e/i)) {
+      if (description.match(/ a/i)) {
+        return Model[make].A
+      }
+      if (description.match(/ c/i)) {
+        return Model[make].C
+      }
+      if (description.match(/ e/i)) {
```

Comment:
Not sure if this is the best approach but it is difficult to just match a single character. Alternatively, we could optionally add `-class` or use `\b`.

Reply:
I think it would make sense to use `\b` as we don't want to match on `Mercedes Enyaq` (if that ever were to be created).